src/daft-core/src/array/ops/list.rs

```diff
@@ -303,3 +335,44 @@ impl FixedSizeListArray {
         }
     }
 }
+
+macro_rules! impl_aggs_list_array {
+    ($la:ident) => {
+        impl $la {
+            fn agg_helper<T>(&self, op: T) -> DaftResult<Series>
+            where
+                T: Fn(&Series) -> DaftResult<Series>,
+            {
+                // Assumes `op`` returns a null Series given an empty Series
+                let aggs = self
+                    .iter()
+                    .map(|s| s.unwrap_or(Series::empty("", self.child_data_type())))
+                    .map(|s| op(&s))
+                    .collect::<DaftResult<Vec<_>>>()?;
```

Comment:
we probably want to have a concat that takes in an iterator of Series for this. materializing all the Series then concating it is going to be slow.

Reply:
tabled until we create an ArrayBuilder so that we don't have to fully materialize the Series objects to concat